util/dvsim/utils.py

```diff
@@ -350,3 +350,28 @@ def color_cell(cell, cclass, indicator="%"):
         for item in subst_list:
             text = text.replace(item, subst_list[item])
     return text
+
+
+def print_msg_list(msg_list_title, msg_list, max_msg_count=-1):
+    '''This function prints a list of messages to Markdown. The argument
+    msg_list_title contains a string for the list title, whereas the msg_list
+    argument contains the actual list of message strings. max_msg_count limits the
+    number of messages to be printed (set to negative number to print all messages).
```

Comment:
The usual format for Python docstrings is a single sentence on a line, then a newline, then the body text (like Git commit messages). So I think you want a `\n\n` before "The argument...".

Reply:
Thanks for the hint - let me change that. It looks like we've in general not followed that convention for docstrings in dvsim, hence we should probably clean this up at some point.